packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -163,19 +164,20 @@ test(testCase, async (t: Test) => {
         gas: 1000000,
       };
       await apiClient.deployContractV1(
-        (parameters as any) as DeployContractV1Request,
+        (parameters as unknown) as DeployContractV1Request,
       );
-    } catch (e) {
+    } catch (err) {
+      const e = err as AxiosError<{ path: string }[]>;
       t2.equal(
-        e.response.status,
+        e?.response?.status,
         400,
         `Endpoint ${fDeploy} without required keychainId: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e?.response?.data.map((param) =>
```

Comment:
@aldousalvarez

Casting the whole exception is also a good solution, but in the interest of keeing the diff smaller on account of conflicts that I foresee with other PRs, please fix the linter warning with a one liner like this: 

```typescript
const fields = e.response.data.map((param: { path: string }) =>
```